calcom/cal.com
# V2 Settings - Security View

## What does this PR do?



This PR adds two screens to the security section of the settings for V2.
- Change password
- Enable 2FA

Note that the settings shell is still a WIP so the pages will be missing headers.

Fixes #3784 



**Environment**: Staging(main branch) / Production

## Type of change




## How should this be tested?



- Visit http://localhost:3000/v2/settings/security/password
	 - Enter an invalid password, an error should appear
 	- Enter a valid password, it should be changed

- Visit http://localhost:3000/v2/settings/security/two-factor-auth
	- Enable 2FA
	- Disable 2FA

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
